source/_docs/guides/drupal8-commandline.md

```diff
@@ -0,0 +1,376 @@
+In a real-world scenario, your content editors may have added nodes and files in the Live environment.
+You would want those updates present on the Test environment with your deployed code.
+The `--updatedb` flag runs Drupal's database update script.
+And `--cc` clears caches.
+The `--note` flag adds a message that is tied to the record of the deployment.
+Under the hood, Pantheon creates a git tag for each deployment.
+This note field is used as an annotation on the git tag.
+To see the tag that was just created.
+
+#### Import the configuration on the Test environment
+
+With the `yml` configuration files now present on the Test environment, they can be imported to the database there:
+
+`terminus drush $TERMINUS_SITE.test -- config-import -y`
+
+#### Sign into Drupal in the Test environment to see the enabled modules.
+
+`terminus drush $TERMINUS_SITE.test -- user-login`
+
+#### Sign into Drupal in the Live environment to see that the modules aren't there yet.
+
+`terminus drush $TERMINUS_SITE.live -- user-login`
+
+Now that you are signed into all three environments you should be seeing the development footer in Dev and Test but not Live.
+
+You can also see the difference between the environments with other Drush commands.
+The `print-module-list` command and `help` command when run on Test will show that Devel modules are enabled and have added even more Drush commands.
+
+`terminus drush $TERMINUS_SITE.test -- pm-list`
+
+`terminus drush $TERMINUS_SITE.test -- help`
+
+On Live you won't see those modules or commands yet.
+
+`terminus drush $TERMINUS_SITE.live -- pm-list`
+
+`terminus drush $TERMINUS_SITE.live -- help`
+
+
+#### Deploy the changes to the Live environment
+
+`terminus env:deploy $TERMINUS_SITE.live --updatedb --cc  --note="Deploying exported config to enable modules"`
+
+We don't need the `--sync-content` flag when going to the Live environment because that environment already has our canonical database.
+
+#### Import the configuration on the Live environment
+
+`terminus drush $TERMINUS_SITE.live -- config-import -y`
+
+Once this command completes you will be able to refresh the Live environment in your browser and see the development footer.
+
+## Pulling content changes down from the Live environment to the Dev environment
+
+In the lifecycle of managing a site, you can expect content editors to add new material to the Live environment.
+You may need this fresh command in your Development environment.
+
+#### Make content on the Live environment
+
+Since you are signed into the Live environment, you can manually make content by browsing to `/node/add`.
+We can also use a Drush command from `devel_generate` module to create some number of nodes.
+
+`terminus drush $TERMINUS_SITE.live  -- generate-content 25`
+
+Including "25" at the end of that command gives us 25 nodes.
+
+#### Bring down the database and media files from the Liv Environment to the Dev environment
```

Comment:
How about "Copy"? "Bring down" can be misconstrued as something you really wouldn't want to do. 😄